extensions/amp-carousel/0.2/test/test-type-slides.js

```diff
@@ -280,15 +280,21 @@ describes.realWin(
         expect(eventSpy).to.have.not.been.called;
       });
 
-      it('should dispatch when changing slides', async () => {
-        const eventSpy = env.sandbox.spy();
-        container.addEventListener('slideChange', eventSpy);
+      it('should dispatch event with index and actionTrust when changing slides', async () => {
+        let event;
+        let counter = 0;
+        container.addEventListener('slideChange', (e) => {
+          event = e;
+          counter++;
+        });
         const carousel = await getCarousel({loop: false});
 
         carousel.implementation_.interactionNext();
         await afterIndexUpdate(carousel);
 
-        expect(eventSpy).to.have.been.calledOnce;
+        expect(counter).to.equal(1);
+        expect(event.data.index).to.eq(1);
```

Comment:
FMI: What's the difference between `to.equal` and `to.eq`?

Reply:
`.eq` is an alias for `.equal`. My VS Code autocompletes to `eq` somehow and I like it since its shorter 🙂

Reply:
nit: Let's actually make these two `equal` then to match the rest of the doc stylistically if they are the same.

Reply:
Done.